esper/controllers/device/device.py

```diff
@@ -143,46 +159,63 @@ def list(self):
                 'id': "ID",
                 'name': "NAME",
                 'model': "MODEL",
-                'state': "CURRENT STATE"
+                'state': "CURRENT STATE",
+                'tags': "TAGS"
             }
 
             for device in response.results:
                 current_state = DeviceState(device.status).name
+                name = device.device_name
+                if device.alias_name and not device.alias_name == '':
```

Comment:
if `device.alias_name` should automatically check for empty strings as well.

Reply:
Done!